model_inference_triage_pipeline/run_model_inference.py

```diff
@@ -54,6 +55,9 @@ def main():
         s3_upload=S3_UPLOAD,
     )
 
+    # Save data to database using api server
+    save_data_to_db(start_time, end_time, CVE_MODEL_TYPE, S3_UPLOAD, ECOSYSTEM)
```

Comment:
Can you make this configurable through the template so it can be turned off with a flag? For testing and debugging purposes so that this service can be run standalone without setting up everything like currently we have to do in the dependency analytics dev cluster setup.

Reply:
Sure will ad param in Template and make change.

Reply:
done